dao/OrderRepository.go

```diff
@@ -215,4 +214,24 @@ func (r *OrderPostgres) AssigningOrderToCourierInDB(order Order) error {
 		return err
 	}
 	return nil
-}
\ No newline at end of file
+}
+
+func (r *OrderPostgres) GetDetailedOrdersByIdFromDB(Id int) (DetailedOrder, error) {
+	var order DetailedOrder
+	transaction, err := r.db.Begin()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
please don't panic except of really non-recoverable situations. Return error from the function